### ISSUE TITLE
[feature]: Pre-fill file name and extension when downloading a file.

### ISSUE THREAD

**Issue by OssamaSijbesma** [hoppscotch/hoppscotch#4265]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

Hi team,

<img width="309" alt="image" src="https://github.com/user-attachments/assets/1c5b874f-a9db-4a6c-bd97-2f862418d000">

When downloading files, the API client currently doesn't pre-fill the name and extension type of the file.

It would greatly improve my user experience if there would be a "send + download" button which would send the payload and download the response immediately with the name and extension of the file pre-filled.

Optionally you could have a look at how other API clients do this.

### Why should this be worked on?

Basically user experience:

The flow is currently: click send, wait for response, click download, change name, change filetype, save file. 😤

While it could be: click send + download, wait for response, and save file. 😌

**Comment by liyasthomas** [comment]
Closing this ticket as the fix is now available in the recent version. Please feel free to reach out if you have any other concerns.
